src/api/db/migrate/20161122121211_move_comments_data.rb

```diff
@@ -3,11 +3,14 @@ def change
     ActiveRecord::Base.transaction do
       Comment.inheritance_column = nil
       Comment.find_each(batch_size: 5000) do |comment|
+        next unless comment
         /\AComment(?<type>.+)/ =~ comment.type
+        type = 'BsRequest' if type == 'Request'
         raise "Uncorrected type for comment with id #{comment.id}" unless ['Project', 'Package', 'BsRequest'].include? type
         comment.commentable_type = type
-        comment.commentable_id = comment.project_id || comment.package_id || comment.bs_request_id
         comment.type = comment.project_id = comment.package_id = comment.bs_request_id = nil
+        comment.commentable_id = comment.project_id || comment.package_id || comment.bs_request_id
+        next unless comment.commentable_id
```

Comment:
because you were setting it to `nil` in the previous line. But I would say that this is something that shouldn't happen and that the migration should fail.